package.json

```diff
@@ -199,7 +199,7 @@
     "babel-plugin-styled-components": "^1.10.7",
     "babel-plugin-transform-remove-console": "^6.9.4",
     "detox": "^15.4.2",
-    "eslint": "^6.6.0",
```

Comment:
and maybe changes from package.json? Seems to be not relevant to your PR

Reply:
yup ill delete the `eslint` related changes to `package.json`. i had made these changes before https://github.com/rainbow-me/rainbow/pull/600 had existed / gotten merged.

Reply:
I was reffing all changes in package.json. Can you move it to another PR if there're important?

Reply:
no they are relevant to this PR.
i made changes to `react-coin-icon` and `react-native-shadow-stack` which match the changes made in this PR. 

shadow stack changes = https://github.com/rainbow-me/react-native-shadow-stack/commit/50b12863d0ce2b164c414fa69280202f6b67d3e7

coin icon changes = https://github.com/mikedemarais/react-coin-icon/commit/d5f1c7f18e79234291e075058d0c99c12e0abeaf#diff-73155c59cb398e7816c5babec357fa01